server_development/topics/providers.adoc

```diff
@@ -308,9 +308,56 @@ public class EjbExampleUserStorageProviderFactory
 
 {project_name} has the ability to execute scripts during runtime in order to allow administrators to customize specific functionalities:
```

Comment:
{project_name} can execute scripts at runtime, allowing administrators to customize the following functionalities:

Reply:
This part already exists here, so I'm not updating it as part of this PR